src/SharpLearning.InputOutput.Test/Csv/CsvRowExtensionsTest.cs

```diff
@@ -125,5 +126,105 @@ public void CsvRowExtensions_Write()
             var actual = writer.ToString();
             Assert.AreEqual(ExpectedWrite, actual);
         }
+
+        [TestMethod]
+        public void CsvRowExtensions_MergeFilesUsingKey_KeepRepeatedColumns()
```

Comment:
I know this is something left over from an earlier refactoring, but could you rename the test methods to be contain KeyCombine instead of MergeFilesUsingKey to indicate that this is the method being tested? For instance `CsvRowExtensions_MergeFilesUsingKey_KeepRepeatedColumns` would become `CsvRowExtensions_KeyCombine_KeepRepeatedColumns`

Reply:
@andjor Besides the two comments in the review, you will also have to sync with the master again, since an update has been added in the mean time. The addition to the master is a change back to unified assembly version across the projects. This will also make it easier to update the version, since you only have to do it one place.

When you have synced with the master, you can update the assembly version of the project in [Directory.Build.props](https://github.com/mdabros/SharpLearning/blob/master/src/Directory.Build.props). Update to version 0.26.3.0. All 3 fields in the file should be updated.